proposals/i18n-system-languages.md

```diff
@@ -67,11 +70,14 @@ It would follow the following signature for [i18n.json](https://chromium.googles
   "name": "getSystemUILanguage",
   "type": "function",
   "nocompile": true,
-  "description": "Gets the current UI language of the Operating System. This is different from $(ref:i18n.getUILanguage) which returns the UI language of the web browser.",
+  "description": "Gets the current UI locale of the Operating System. This is different from $(ref:i18n.getUILanguage) which returns the UI locale of the web browser. This locale could include additional information like preferred datetime format and number system if supported by the OS.",
```

Comment:
Is the "additional information" part a standard? I know Apple platforms can include it, but that isn't something we have had in web extensions before.

Reply:
@xeenon yes, this is specified by unicode. This was discussed during the shared session with the WECG/I18NWG meeting at TPAC. However, giving a further look at this I see how this should not be part of the getSystemUILanguage. Updated the PR to reflect this. Still, providing this information can be useful, but it should be implemented using some other method / proposal.